app/routes/planet_routes.py

```diff
@@ -0,0 +1,49 @@
+from flask import Blueprint, jsonify, abort, make_response
+
+class Planet:
+    def __init__(self, id, name, description, has_moon=None):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.has_moon = has_moon
+
+    def make_dict(self):
+        return dict(
+                id = self.id,
+                name = self.name,
+                description = self.description,
+                has_moon = self.has_moon,  
+            )
+
+planets = [
+    Planet(1, "Mercury", "terrestrial", False),
+    Planet(2, "Jupiter", "gaseous", True),
+    Planet(3, "Earth", "terrestrial", True)
+]
```

Comment:
We'll be getting rid of this collection soon, but consider using named arguments here to help this be a little more self-documenting about what the various values are. Id and name are pretty clear, but it might be harder to tell that the second string is a description, and I'd be hard pressed to guess that the final boolean was about whether there was a moon there.